tests/unit/core/api/test_abuse.js

```diff
@@ -0,0 +1,91 @@
+import * as api from 'core/api';
+import { reportAddon } from 'core/api/abuse';
+import {
+  dispatchClientMetadata,
+  dispatchSignInActions,
+  fakeAddon,
+} from 'tests/unit/amo/helpers';
+import {
+  createApiResponse,
+  createFakeAddonAbuseReport,
+} from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  let mockApi;
+
+  beforeEach(() => {
+    mockApi = sinon.mock(api);
+  });
+
+  describe('reportAddon', () => {
+    function _reportAddon(extraArguments = {}) {
```

Comment:
This helper doesn't seem to be helping. I'd suggest using `reportAddon()` directly instead.

Reply:
Hah, yeah, true. It used to help! 😉